src/sdk/controllers/userApps.ts

```diff
@@ -273,14 +290,30 @@ export class UserApps {
 
         const base64UrlPayload = params.get('payload');
 
-        if (!base64UrlPayload) throwError("payload parameter doesn't exists", SdkErrors.MissingParams);
+        if (!base64UrlPayload) throwError("payload parameter doesn't exist", SdkErrors.MissingParams);
 
         const parsedPayload = base64UrlToObj(base64UrlPayload);
 
         if (!parsedPayload || !parsedPayload.requests)
             throwError('No requests found in payload', SdkErrors.MissingParams);
 
-        const loginRequests = parsedPayload.requests.map((r: string) => new LoginRequest(r));
+        // const loginRequests = parsedPayload.requests.map((r: string) => new LoginRequest(r));
+        parsedPayload.requests = parsedPayload.requests.filter((request: string) => request !== null);
```

Comment:
should this be done above L297?

Reply:
that would then catch an edge case where `parsedPayload.requests` contained an array of null objects

Reply:
that would then catch an edge case where `parsedPayload.requests` contained an array of null objects